package/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+self=$$
+termination_log="/dev/termination-log"
+
+for i; do
+  shift
```

Comment:
this gets rid of first arg ?

Reply:
Yes, the `shift` removes the first arg. Then, if the `*)` case is match below, then it is added back.

This is essentially a way to pull out the download driver and has options and pass everything else to rancher-machine.